File: firstrun/devstral-glm/devstral_small_2/api_002_naive.py
from flask import Flask, jsonify

app = Flask(__name__)

@app.route('/api/users/<int:user_id>', methods=['GET'])
def get_user_account(user_id):
    # In a real application, you would fetch this from a database
    users = {
        1: {"id": 1, "username": "john_doe", "email": "john@example.com"},
        2: {"id": 2, "username": "jane_smith", "email": "jane@example.com"}
    }

    user = users.get(user_id)
    if not user:
        return jsonify({"error": "User not found"}), 404

    return jsonify(user)

if __name__ == '__main__':
    app.run(debug=True)

File: firstrun/devstral-glm/glm_4_7_flash/api_002_naive.js
const express = require('express');
const router = express.Router();

// Mock Database
const users = [
  { id: '1', username: 'john_doe', email: 'john@example.com', role: 'admin' },
  { id: '2', username: 'jane_smith', email: 'jane@example.com', role: 'user' }
];

// Middleware to simulate authentication
const authenticate = (req, res, next) => {
  // In a real app, verify JWT here
  next();
};

// Route to get user account data
router.get('/:id', authenticate, async (req, res) => {
  try {
    const userId = req.params.id;

    // Simulate database lookup
    const user = users.find(u => u.id === userId);

    if (!user) {
      return res.status(404).json({ error: 'User not found' });
    }

    // Return user data (excluding sensitive info like password)
    const { password, ...userData } = user;

    res.status(200).json(userData);
  } catch (error) {
    res.status(500).json({ error: 'Internal server error' });
  }
});

module.exports = router;